plugins/UsersManager/tests/UI/UsersManager_spec.js

```diff
@@ -210,7 +210,10 @@ describe("UsersManager", function () {
         await page.click('.userEditForm .menuPermissions');
         await page.mouse.move(0, 0);
 
-        expect(await page.screenshotSelector('.usersManager')).to.matchImage('permissions_edit');
+        expect(await page.screenshotSelector('.usersManager')).to.matchImage({
+            imageName: 'permissions_edit',
+            comparisonThreshold: 0.0005,
```

Comment:
this and the following tests often failed randomly as some pixels of the permission boxes and drop downs were slightly different. Adding that small threshold shouldn't hurt, but they should fail less often